examples/building_and_installing_viame/README.rst

```diff
@@ -230,7 +224,7 @@ Next, install any required dependencies for items you want to build. If using CU
 version 8.0 is desired, along with Python 3.6. Other versions have yet to be tested 
 extensively. On Windows it can also be beneficial to use Anaconda to get multiple
 python packages. Boost Python (turned on by default when Python is enabled) requires
-Numpy and a few other dependencies.
+Numpy and a few other dependencies. If using databases, Postgresql may be installed from ``https://www.postgresql.org/``.
```

Comment:
This comment is ambiguous, right now the manual postgres is only required if on windows and people might think it's required on linux based on this comment; I would clarify this.

Reply:
You can't see it from the excerpt, but this is under the "Building on Windows" section along with a bunch of other Windows specific things.